.github/workflows/ubuntu.yml

```diff
@@ -1,10 +1,12 @@
 name: Ubuntu Workflow
 
 on:
-  pull_request:
-    branches: [ gcos4gnucobol-3.x ]
+  # pull_request:
+  #   branches: [ gcos4gnucobol-3.x ]
   push:
-    branches: [ gcos4gnucobol-3.x ]
+    branches: [ fix-ci ]
```

Comment:
you likely want to replace that branch name or- if it is possible - don't limit it with a branch name; if you want the CI only be triggered manually or on pull - adjust the one above (I just did not had that and as I've only adjusted the CI used the push)

Reply:
... then maybe add a changelog for coverage (if my change was fine then take me, otherwise you as responsible)

Reply:
Yes sure :smile:… just trying to re-enable workflows for macos and windows beforehand.

Reply:
Somehow some changes I made broke that…